hack/cherry-pick-pull.sh

```diff
@@ -200,6 +208,10 @@ for pull in "${PULLS[@]}"; do
 
   # remove the patch file from /tmp
   rm -f "/tmp/${pull}.patch"
+
+  # get the release notes
+  notes=$(curl "https://api.github.com/repos/${MAIN_REPO_ORG}/${MAIN_REPO_NAME}/pulls/${pull}" -sS | jq '.body' | grep -Po "\`\`\` *${RELEASE_NOTE_CATEGORY} ${RELEASE_NOTE_TARGET_GROUP}.*?\`\`\`")
```

Comment:
Hm, just a question. Does this regex always require an empty space between "```" and the release category?

Reply:
I tested it with release notes that have and have not a space between the back ticks and the category, in both cases it was working. The release-note functionality in cc-utils also [have the space](https://github.com/gardener/cc-utils/blob/4df37b6c592329208bc8a0f02f00d9b864aa4f98/github/release_notes/util.py#L414-L419), thus I decided to keep it.